hyper_click_command.py

```diff
@@ -71,7 +71,13 @@ def is_valid_line(self, line_content):
 
     def is_visible(self, event=None):
         view = self.view
+
+        # old style discouraged single-string selector
         selector = self.settings.get('selector')
+        if not selector:
+            # new style list of strings selectors
+            selector = "(" + ")|(".join(self.settings.get('selectors')) + ")"
```

Comment:
```suggestion
        selector = "(" + ")|(".join(self.settings.get('selectors')) + ")"
```
Since the single-string selector was only publicly available on 2.0.0, I think we can get away with skipping this check